WasatchNET/Driver.cs

```diff
@@ -271,54 +271,58 @@ void logDevice(UsbRegistry usbRegistry)
             foreach (string line in deviceInfoSummaries)
                 logger.debug("Summary: {0}", line);
 
-            foreach (UsbConfigInfo cfgInfo in device.Configs)
+            // YOU ARE HERE -- does this die on Zynq? On ARM in general?

```

Comment:
I need to do testing with more ARM spectrometers to figure out what's going on here.  However, for now I'm disabling this metadata dump as it's hanging on the SiG-VIS.